scripts/proto-gen.sh

```diff
@@ -23,15 +13,11 @@ for i in "${GOPATH}/src/${PROTO_SRC}"/*; do
 	fi
 
 	if ls $i/*.proto > /dev/null 2>&1; then
-		proto_files=$(ls $i/*.proto | sed -e "s@${GOPATH}@@g")
+		proto_files=$(ls $i/*.proto | sed -e "s@${GOPATH}@${GOPATH}@g")
 		echo "generating from ${proto_files}"
-		# need the additional m3db_path mount in docker because it's a symlink on the CI.
-		m3db_path=$(realpath $GOPATH/src/github.com/m3db/m3)
     resolve_protos="Mgoogle/protobuf/timestamp.proto=github.com/gogo/protobuf/types,Mgoogle/protobuf/wrappers.proto=github.com/gogo/protobuf/types,Mgoogle/protobuf/any.proto=github.com/gogo/protobuf/types,Mgoogle/protobuf/struct.proto=github.com/gogo/protobuf/types"
 
-		docker run --platform linux/amd64 --rm -w /src -v $GOPATH/src:/src -v ${m3db_path}:/src/github.com/m3db/m3 \
-		$UID_FLAGS $PROTOC_IMAGE_VERSION \
-			 --gogofaster_out=${resolve_protos},plugins=grpc:/src \
-			 -I/src -I/src/github.com/m3db/m3/vendor ${proto_files}
+	  protoc --gogofaster_out=${resolve_protos},plugins=grpc:${GOPATH}/src \
```

Comment:
I think we should try to stick with the containerized protobuf here ideally--we want to use a consistent protobuf version, rather than whatever the developer has "lying around".